app/views/errors/internal_server_error.html.erb

```diff
@@ -1,6 +1,37 @@
-<% content_for :title, "Internal Server Error" %>
+<% content_for :title, 'Internal Server Error' %>
 
-<% content_for :error_message do %>
-  <p>The server encountered a configuration or processing error and was unable to complete your request. Please try
-    again, and if the problem persists contact the administrator.</p>
+<h1 class="error-code has-color-red-500">500</h1>
+<h2 class="error-name has-color-red-400">server error</h2>
+
+<p class="is-lead">
+  <% if Rails.env.production? %>
+    It's not your fault.
+  <% else %>
+    It's totally your fault. Fix it.
```

Comment:
*laugh*  Uh, let's make sure this can never leak out to a user... with luck that is-production check does that. :-)

Reply:
If it's in development or test mode in a production setting... we already have much bigger problems than this :)